src/osemgrep/core/Metrics_.ml

```diff
@@ -290,6 +290,7 @@ let add_engine_type (engine_type : Engine_type.t) =
           secrets_config;
           code_config;
           supply_chain_config;
+          _;
```

Comment:
@liukatkat please avoid those patterns; they prevent us next time we add a field to get the opportunity
to visit this code and check whether it needs to be updated.